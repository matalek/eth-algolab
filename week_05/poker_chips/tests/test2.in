30
4
11 9 12 9
3 34 60 38 86 54 54 7 31 59 41
24 31 17 14 16 59 13 87 33
41 70 72 36 96 90 16 21 75 12 73 53
90 87 95 27 9 50 55 43 25
4
11 10 8 8
6 8 91 87 75 17 78 81 65 29 31
27 20 72 13 89 57 100 27 46 59
79 91 74 73 48 36 49 57
94 49 76 24 16 62 14 34
3
7 11 7
22 99 32 88 21 24 94
75 51 52 61 54 84 15 62 50 74 15
41 20 49 78 51 86 51
5
5 4 7 10 12
93 71 76 71 29
12 95 49 84
65 82 6 75 15 92 70
90 90 13 75 38 8 27 82 33 67
73 30 74 62 47 41 77 35 25 1 91 11
4
9 8 8 6
82 53 99 59 71 13 93 77 47
97 34 18 53 85 94 89 85
27 5 100 82 60 54 35 24
49 63 57 61 83 30
5
11 3 2 7 8
88 100 77 36 48 15 2 21 17 41 33
75 28 8
89 8
90 26 100 83 63 87 29
91 96 17 52 5 53 84 66
5
8 6 10 8 9
76 14 1 55 61 84 85 55
98 92 79 63 77 51
47 26 69 27 81 27 61 50 92 86
51 100 29 93 80 7 38 90
57 14 75 15 35 19 70 96 75
3
11 10 12
54 88 79 57 17 10 26 62 30 81 91
61 48 38 30 7 2 89 16 20 32
89 83 84 45 66 5 73 32 3 71 78 76
3
7 12 11
6 12 35 13 74 95 10
86 34 79 57 23 95 23 32 38 33 77 42
29 100 16 9 44 22 25 92 38 10 52
2
9 12
88 36 68 66 34 100 51 7 25
81 85 13 98 16 52 5 53 70 92 32 24
3
7 7 10
1 1 1 1 1 1 2
1 1 2 2 2 1 1
2 1 2 1 1 1 2 1 2 2
5
12 4 12 7 8
1 2 2 1 2 2 2 1 1 1 1 2
2 2 1 2
2 1 1 2 1 1 1 1 1 1 1 1
2 1 2 2 2 1 2
2 2 1 1 2 2 2 2
5
4 9 6 8 5
1 1 1 2
2 2 1 1 2 2 2 1 1
2 2 2 2 2 2
1 1 1 1 2 1 2 2
2 2 2 2 1
3
12 9 7
1 2 1 2 1 2 1 1 2 1 1 2
2 1 1 2 2 2 2 1 1
1 1 2 2 2 1 1
3
7 6 7
2 2 2 1 2 2 2
2 2 1 2 1 1
2 1 2 1 2 2 1
3
10 9 11
1 2 1 2 2 1 2 2 1 2
2 1 1 2 2 1 2 1 2
1 2 2 2 1 1 2 1 1 2 1
2
8 10
1 2 1 2 1 1 1 2
1 2 2 2 1 2 1 1 2 1
5
9 4 11 6 8
2 2 1 2 1 2 1 2 2
2 2 2 2
1 1 1 2 1 1 2 2 2 2 1
2 1 1 2 1 2
1 2 2 2 1 2 2 1
2
11 7
2 2 1 1 1 2 2 1 1 1 2
2 2 2 2 2 2 2
5
3 10 9 4 8
1 2 2
1 1 1 1 1 1 2 1 2 1
2 2 1 1 1 2 2 1 1
2 2 1 2
2 1 2 2 1 1 2 2
3
6 9 11
5 8 3 10 3 3
3 10 8 3 3 7 5 10 3
5 8 3 10 3 3 10 3 10 8 7
2
7 7
7 6 6 4 4 4 4
6 7 4 4 6 4 4
5
5 10 4 12 11
1 1 11 1 12
1 1 11 1 12 6 9 7 5 9
1 1 11 1
1 1 11 1 12 6 9 7 5 9 8 8
1 1 11 1 12 6 9 7 5 9 8
5
9 6 6 11 9
6 8 1 2 7 11 7 2 5
6 8 1 2 7 11
6 8 1 2 7 11
6 8 1 2 7 11 7 2 5 9 11
6 8 1 2 7 11 7 2 5
3
10 10 11
4 3 1 6 2 5 10 5 4 6
2 6 8 5 4 1 10 5 3 4
4 3 1 6 2 5 10 5 4 6 8
3
11 11 7
3 5 6 1 10 10 8 9 3 7 9
1 9 3 9 10 6 5 8 10 3 7
3 5 6 1 10 10 8
4
8 9 6 9
6 6 6 7 8 2 5 6
6 6 6 7 8 2 5 6 5
6 6 5 7 5 8
6 6 6 7 8 2 5 6 5
5
6 7 6 12 8
10 6 9 5 10 1
10 6 9 5 10 1 6
5 3 6 10 9 3
10 6 9 5 10 1 6 3 10 3 6 3
10 6 9 5 10 1 6 3
2
9 12
5 7 3 12 5 1 4 3 1
5 7 3 12 5 1 4 3 1 6 3 6
4
6 8 12 8
5 7 2 9 11 10
11 3 7 2 5 9 5 2
5 7 2 9 11 10 2 5 3 2 11 5
5 7 2 9 11 10 2 5
