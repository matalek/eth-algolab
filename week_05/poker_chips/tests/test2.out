5
3
1
1
2
3
4
3
3
0
11
38
25
14
11
16
7
37
5
29
12
5
47
54
15
12
25
36
9
22
